### PR TITLE
Update announce bar for 2.10.7

### PR DESCRIPTION
**Pull request by serinamarie**
### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [ ] This pull request includes tests or only affects documentation.
- [ ] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.

### CODE REVIEW

File: docs/overrides/main.html [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 {% block announce %}
 <div style="color: white;">
-  <a href="https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md#release-2106" style="color: #07e798;">
-    <strong>Prefect 2.10.6 is here!</strong>
-  </a> With multideployment project configuration, improved run restart handling, and
-  <a href="https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md#release-2106" style="color: #07e798;">
+  <a href="https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md#release-2107" style="color: #07e798;">
+    <strong>Prefect 2.10.7 is here!</strong>
+  </a> With a new and improved Flows page, and `on_crashed` state change hook for flows, and
```
Comment (zhen0) [inline docs/overrides/main.html:7]:
```suggestion
  </a> With a new and improved flows page, and `on_crashed` state change hook for flows, and
```

For consistency with the release notes lets keep this lower case.

**Review by zhen0 — COMMENTED** [review]
One tiny tiny change but otherwise LGTM!

Thanks so much @serinamarie

File: docs/overrides/main.html (hunk 1 above)
Comment (serinamarie) [reply docs/overrides/main.html:7]: The release notes use a capital F, should that be updated, then?

https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md#new-and-improved-flows-page

Comment (serinamarie) [reply docs/overrides/main.html:7]: If so, I can update it

Comment (serinamarie) [reply docs/overrides/main.html:7]: @zhen0 If so, I can update it

Comment (zhen0) [reply docs/overrides/main.html:7]: hmm... the release notes are actually a bit inconsistent now.  They use F in the title but not in the text.  But it's a tiny detail.  Ignore my comment as yours is the best match and follows Bill's original use of F.

**Review by zhen0 — APPROVED** [review]

**Pull request merged**